sentry.client.config.js

```diff
@@ -0,0 +1,17 @@
+// This file configures the initialization of Sentry on the browser.
+// The config you add here will be used whenever a page is visited.
+// https://docs.sentry.io/platforms/javascript/guides/nextjs/
+
+import * as Sentry from '@sentry/nextjs';
+
+const SENTRY_DSN = process.env.SENTRY_DSN || process.env.NEXT_PUBLIC_SENTRY_DSN;
+
+Sentry.init({
+  dsn: SENTRY_DSN || 'https://ee0783ce357641a3bd5e692f5f3aaa21@o1107338.ingest.sentry.io/6134404',
```

Comment:
Not familiar with this...this URL doesn't have to be a secret?

Reply:
nope, its client side, the client can't be trusted, blah, blah, blah.

Technically, if someone wanted to be malicious they should start sending us fake errors, but thats true with any client side web tracking.